### PR TITLE
Making HL tests Api Compatible with release/v1.16

### PR DESCRIPTION
**Pull request by abukhoy**

### CODE REVIEW

**Review by quic-rishinr — APPROVED** [review]

File: tests/cloud/conftest.py [hunk 1]
```diff
@@ -58,15 +59,17 @@ def __init__(self,model_name,num_cores,prompt,prompts_txt_file_path,aic_enable_d
...
         self.mxint8 = mxint8
+        self.full_batch_size = None
         self.device_group = device_group
 
     def model_card_dir(self):
         return str(os.path.join(QEFF_MODELS_DIR, str(self.model_name)))
     
     def qpc_base_dir_name(self):
-        return get_qpc_dir_name_infer(self.num_cores, self.mos, self.batch_size, self.prompt_len, self.ctx_len, self.mxfp6, self.mxint8, self.device_group)
+        return get_qpc_dir_name_infer(self.num_cores, self.mos, self.batch_size, self.prompt_len, self.ctx_len, self.mxfp6, self.mxint8, self.device_group, self.full_batch_size)
```
Comment (vbaddi) [inline tests/cloud/conftest.py:72]: is pre-commit linter run on it? looks like code is not formatted. pls run the pre commit.

**Review by vbaddi — CHANGES_REQUESTED** [review]

File: tests/cloud/conftest.py [hunk 2]
```diff
@@ -58,15 +59,17 @@ def __init__(self,model_name,num_cores,prompt,prompts_txt_file_path,aic_enable_d
         self.batch_size = batch_size
         self.prompt_len = prompt_len
         self.ctx_len = ctx_len
+        self.generation_len = None
         self.mxfp6 = mxfp6
         self.mxint8 = mxint8
+        self.full_batch_size = None
```
Comment (vbaddi) [inline tests/cloud/conftest.py:65]: Can you enable one test with full_batch_size, current supported models with this flags are llama, mistral, mixtral.
So for the tiny llama card you can pass full_batch_size and get the test verified.

File: tests/cloud/conftest.py (hunk 1 above)
Comment (abukhoy) [reply tests/cloud/conftest.py:72]: It is done.

File: tests/cloud/conftest.py (hunk 2 above)
Comment (abukhoy) [reply tests/cloud/conftest.py:65]: Doing it now.

**Comment by vbaddi** [comment]
@abukhoy pls rebase your PR to the release branch, latest changes are missing, CI is failing.

**Comment by abukhoy** [comment]
@vbaddi CI is passed.

**Review by vbaddi — APPROVED** [review]
LGTM. Thanks

**Review by quic-rishinr — APPROVED** [review]

**Comment by vbaddi** [comment]
cc @anujgupt-github

**Comment by vbaddi** [comment]
cc @irajagop could you pls review this pr?

**Review by irajagop — APPROVED** [review]

**Comment by ochougul** [comment]
Closing this, to declutter.

**Pull request closed without merging**